experimental/origin-graphql/src/mutations/_txHelper.js

```diff
@@ -77,7 +77,10 @@ export default function txHelper({
         })
         if (contracts.automine) {
           setTimeout(() => {
-            contracts.web3Exec.currentProvider.send(
+            // This needs to be sendAsync, because the provider engine used for
+            // the mobile wallet linker client doesn't support synchronous
+            // transactions.
+            contracts.web3.currentProvider.send(
```

Comment:
That's fine, because I changed `web3Exec` to `web3`. The issue we discussed was that the `WalletLinkerClient`'s web3 provider doesn't support synchronous sends. Switching to `contracts.web3` allows us to keep using `send`.